kunai-lib/lib/DEX/DVM/recursive_traversal_disassembler.cpp

```diff
@@ -39,8 +39,6 @@ void RecursiveTraversalDisassembler::disassembly(std::vector<std::uint8_t> &buff
     auto buffer_size = buffer_bytes.size();
     // opcode
     std::uint32_t opcode;
-    // check if the method contains a switch
-    bool exist_switch = false;
```

Comment:
This variable is not used. Should we keep it?

Reply:
It comes from old code, so it's okay removing it